src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private static final Random RANDOM = new Random();

```

Comment:
According to the checklist, don't use static methods in your solution. Static fields and methods can lead to issues with object-oriented design, especially when dealing with state and concurrency. Consider making `RANDOM` an instance variable instead of a static one.